content/en/docs/contributing/release-process.md

```diff
@@ -239,157 +269,154 @@ required only to avoid rate-limits imposed on anonymous API users.
-      `CMREL_RELEASE_NAME` variable is found in the output of the previous
-      command, `cmrel stage`. Look for the line that contains the `gs://` link:
+    1. Set the `CMREL_RELEASE_NAME` variable in your shell. The value for the
+       `CMREL_RELEASE_NAME` variable is found in the output of the previous command,
+       `cmrel stage`. Look for the line that contains the `gs://` link:
 
         ```sh
-         gs://cert-manager-release/stage/gcb/release/v1.3.0-alpha.1-c2c0fdd78131493707050ffa4a7454885d041b08
-                                                     <------------- CMREL_RELEASE_NAME -------------------->
-         ```
+        gs://cert-manager-release/stage/gcb/release/v1.3.0-alpha.1-c2c0fdd78131493707050ffa4a7454885d041b08
+        #                                           <---------- CMREL_RELEASE_NAME ----------------------->
+        ```
 
-        Copy that part into a variable in your shell:
+        Copy that part into a variable in your shell (no need to export it):
 
         ```sh
-        export CMREL_RELEASE_NAME=v1.3.0-alpha.0-77b045d159bd20ce0ec454cd79a5edce9187bdd9
+        CMREL_RELEASE_NAME=v1.3.0-alpha.0-77b045d159bd20ce0ec454cd79a5edce9187bdd9
         ```
 
-   1. Do a `cmrel publish` dry-run to ensure that all the staged resources are
-      valid. Run the following command:
+    2. Do a `cmrel publish` dry-run to ensure that all the staged resources are
+       valid. Run the following command:
 
-        ```bash
+        ```sh
         # Must be run from the "cert-manager/release" repo folder.
         cmrel publish --release-name "$CMREL_RELEASE_NAME"
         ```
 
         You can view the progress by clicking the Google Cloud Build URL in the
         output of this command.
 
-   2. While the build is running, send a third Slack message in reply to
-      the first message:
-
-        {{% pageinfo %}}
-Follow the `cmrel publish` dry-run build: <https://console.cloud.google.com/cloud-build/builds16f6f875-0a23-4fff-b24d-3de0af207463?project=1021342095237>
-        {{% /pageinfo %}}
-
+    3. While the build is running, send a third Slack message in reply to
+       the first message:
 
-   3. Next publish the release artifacts for real.
+        <div class="pageinfo pageinfo-primary"><p>
+        Follow the `cmrel publish` dry-run build: https://console.cloud.google.com/cloud-build/builds16f6f875-0a23-4fff-b24d-3de0af207463?project=1021342095237
+        </p></div>
 
-        If the last step succeeded, you can now re-run the `cmrel publish` with
-        the `--nomock` argument to actually publish the release artifacts to
-        GitHub, `Quay.io`, to our [ChartMuseum](https://charts.jetstack.io)
-        instance, etc.
+    4. Next publish the release artifacts for real. The following command will
+       publish "for real" the artifacts to GitHub, `Quay.io`, to our
+       [ChartMuseum](https://charts.jetstack.io) instance:
```

Comment:
below: `RELEASE_NAME` -> `CMREL_RELEASE_NAME`

Reply:
Thanks, done in dbed8be